clang/test/Driver/sycl-intelfpga-aoco.cpp

```diff
@@ -58,7 +58,7 @@
 // CHK-FPGA-AOCO: spirv-to-ir-wrapper{{.*}} "[[LIBLIST]]" "-o" "[[LIBLIST2:.+\.txt]]"
 // CHK-FPGA-AOCO: llvm-link{{.*}} "-o" "[[LINKEDBC:.+\.bc]]"
 // CHK-FPGA-AOCO: llvm-link{{.*}} "--only-needed" "[[LINKEDBC]]" "@[[LIBLIST2]]" "-o" "[[LINKEDBC2:.+\.bc]]"
-// CHK-FPGA-AOCO: sycl-post-link{{.*}} "-split-esimd"{{.*}} "-O2" "-spec-const=emulation" "-device-globals" "-o" "[[SPLTABLE:.+\.table]]" "[[LINKEDBC2]]"
```

Comment:
Hmm..I thought this will be a NFC. Why are these options getting removed? Thanks

Reply:
Just some cleanup - the options themselves in the context of this test are not important.  Reducing the options passed here allows for less potential changes down the line if the options passed to sycl-post-link change.